Octokit.Reactive/Clients/ObservablePullRequestsClient.cs

```diff
@@ -18,15 +18,17 @@ public class ObservablePullRequestsClient : IObservablePullRequestsClient
         /// <summary>
         /// Client for managing comments.
         /// </summary>
-        public IObservablePullRequestReviewCommentsClient Comment { get; private set; }
+        [Obsolete("Please use ObservablePullRequestsClient.ReviewComment. This will be removed in a future version")]
+        public IObservablePullRequestReviewCommentsClient Comment { get;  set; }
```

Comment:
We still need the `Comment` endpoint to function so let's make it a getter only, and have it return the new client

eg
``` csharp
public IObservablePullRequestReviewCommentsClient Comment { get { return this.ReviewComment; } }
```